docs/modules/ROOT/pages/getting-started.adoc

```diff
@@ -1 +1,162 @@
 = Getting Started
+
+== Introduction
+
+The easiest way to get started using Solidity Hot Loader is using a https://docs.openzeppelin.com/starter-kits/2.3/[Starter Kit]. In this guide we walk through using Solidity Hot Loader with the https://docs.openzeppelin.com/starter-kits/2.3/tutorial[Tutorial Starter Kit]. The Starter Kit includes a Counter dapp, which displays the value of a Counter that can be increased/decreased. We will update the code in the smart contract to show the power of the Solidity Hot Loader.
+
+== Setup
+First we need a local blockchain to develop on. We use `ganache-cli`, so go ahead and install (if you haven't already) and then run a local blockchain using `ganache-cli`
+
+[source,console]
+----
+$ npm install -g ganache-cli
+$ ganache-cli --deterministic
+----
+
+Next we need a project directory to develop our project in. In a separate terminal create the project directory.
+
+[source,console]
+----
+$ mkdir myproject && cd myproject
+----
+
+Unpack the Tutorial Starter Kit. (This could take a little while so maybe make a cup of tea)
```

Comment:
```suggestion
Unpack the Tutorial Starter Kit using the xref:sdk::index.adoc[*OpenZeppelin CLI*]. This could take a little while, so maybe make a cup of tea.
```